tests/integration/ovs_test.py

```diff
@@ -193,6 +194,22 @@ def test_nm_ovs_plugin_missing():
             )
 
 
+def test_ovs_service_missing():
+    with disable_service("openvswitch"):
+        with pytest.raises(NmstateDependencyError):
+            libnmstate.apply(
+                {
+                    Interface.KEY: [
+                        {
+                            Interface.NAME: BRIDGE1,
+                            Interface.TYPE: InterfaceType.OVS_BRIDGE,
+                            Interface.STATE: InterfaceState.UP,
+                        }
+                    ]
+                }
+            )
```

Comment:
If some how `libnmstate.apply()` succeeded(for example the `disable_service()` failed to disable the service), you have a leftover OVS bridge. Please remove it also.

Reply:
I am removing it after at the end. Thanks!